oap-storage-mongo/src/main/java/oap/storage/mongo/Migration.java

```diff
@@ -100,14 +100,6 @@ public String toScript( String mongoHost, int mongoPort, String database, String
         return script.toString();
     }
 
-    public String toScript( String mongoHost, int mongoPort, String database ) {
```

Comment:
since it's used in multiple projects it would be better to add the overload method to keep backward compatibility